commerce-channel-web/src/main/java/com/liferay/commerce/channel/web/internal/display/context/BaseCommerceChannelDisplayContext.java

```diff
@@ -58,10 +58,10 @@ public PortletURL getPortletURL() throws PortalException {
 		return portletURL;
 	}
 
-	protected final CPRequestHelper cpRequestHelper;
 	protected final HttpServletRequest httpServletRequest;
 	protected final LiferayPortletRequest liferayPortletRequest;
 	protected final LiferayPortletResponse liferayPortletResponse;
 	protected final PortalPreferences portalPreferences;
+	protected final CommerceChannelRequestHelper requestHelper;
```

Comment:
`CommerceCatalogRequestHelper requestHelper` should be `CommerceCatalogRequestHelper commerceCatalogRequestHelper`